src/includes/getting-started-primer/dart.mdx

```diff
@@ -1,5 +1,18 @@
 <Note>
 
-Pure Dart SDK used by any Dart application like AngularDart, CLI and Server, it enables reporting messages and errors.
+Sentry's Dart SDK enables automatic reporting of errors, messages, and exceptions. The SDK is available on GitHub [`sentry-dart`](https://github.com/getsentry/sentry-dart/). For Flutter apps, refer to [our Flutter documentation](/platforms/flutter) instead.
 
 </Note>
+
+Features:
+
+- Automatic native crash error tracking (using both [Android](/platforms/android/) and [iOS/macOS](/platforms/apple/)), including
+    - Java, Kotlin, C, and C++ code for Android
+    - ObjC, Swift, and C for iOS and macOS
+- Offline storage of events
+- Events [enriched](/platforms/dart/enriching-events/context/) with device data
+- Breadcrumbs automatically captured:
+    - by the [Dart SDK](/platforms/dart/enriching-events/breadcrumbs/#automatic-breadcrumbs)
+    - with the Native SDKs [Automatic Breadcrumbs for Android](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs) and [Automatic Breadcrumbs for iOS](/platforms/apple/configuration/integrations/default/#sentryautobreadcrumbtrackingintegration)
+    - as well as `http` with the [Dart SDK](/platforms/dart/)
+- [Release Health](/product/releases/health/) tracks crash free users and sessions
```

Comment:
This is Flutter only

Reply:
It seems like you've indicated all the features are Flutter only. Are there any features to list on the getting started primer page for Dart? @bruno-garcia maybe you can answer this?

Reply:
I will double-check and make new suggestions if needed.
Can't use suggestions over the phone, sorry about that.

Reply:
I've done the best I can on this list of features based on what's listed for Android and what shows up as available for Dart. You'll need to finalize this.